src/main/resources/org/nest/sympy/misc.py

```diff
@@ -0,0 +1,178 @@
+from sympy.parsing.sympy_parser import parse_expr
+from sympy import *
+
+from sympy.matrices import zeros
+
+# Define constants
+MAX_ORDER = 10
+MAX_TRIES = 100
+
+# 't' is predefined in NESTML and represents time
+# 'derivative_factor' is the factor in 
+# shape'=derivative_factor*shape in case shape satisfies such an ODE.
+derivative_factor, t = symbols("derivative_factor, t")
+        
+
+class ShapeFunction(object):
+    '''The aim of this script is to check for any given function of
+    positive `t`, which we will call `shape`, if it satisfies a linear
+    homogeneous ODE. If it does, it will give it's order, initial
+    values, the equation itself and a list of all factors of
+    derivatives (which we will call `derivative_factors`) of the shape
+    function in the ODE, i.e. if shape''= a*shape+b*shape' the list
+    will be: [a,b].
+
+    Example:
+    ========
+
+    Convert the postsynaptic response shape from a function form  
+       shape = t * exp(t * a)
+    to an ODE form
+       shape'' = -a**2 * shape + 2*a * shape'
+    '''
+
```

Comment:
I find this comment a bit strange here. The comment here should explain what the class `ShapeFunction` is about. The comment above should rather be a docstring for the module, i.e. go to the top of the file.